src/terrain/util/config.clj

```diff
@@ -24,26 +24,26 @@
   []
   (cc/mask-config props :filters [#"(?:irods)[-.](?:user|pass|key|secret)"]))
 
-(cc/defprop-int listen-port
+(cc/defprop-optint listen-port
   "The port that terrain listens to."
   [props config-valid configs]
-  "terrain.app.listen-port")
+  "terrain.app.listen-port" 60000)
 
-(cc/defprop-str environment-name
+(cc/defprop-optstr environment-name
   "The name of the environment that this instance of Terrain belongs to."
   [props config-valid configs]
-  "terrain.app.environment-name")
+  "terrain.app.environment-name" "docker-compose")
 
-(cc/defprop-vec allowed-groups
+(cc/defprop-optvec allowed-groups
   "The names of the groups that are permitted to access secured admin services."
   [props config-valid configs]
-  "terrain.cas.allowed-groups")
+  "terrain.cas.allowed-groups" ["core-services", "tito-admins", "tito-qa-admins", "dev"])
 
-(cc/defprop-str uid-domain
+(cc/defprop-optstr uid-domain
   "The domain name to append to the user identifier to get the fully qualified
    user identifier."
   [props config-valid configs]
-  "terrain.uid.domain")
+  "terrain.uid.domain" "cyverse.org")
```

Comment:
My thought was that we'd be switching to `cyverse.org` eventually anyway. We can override it for the time being.